charts/linkerd2/values.yaml

```diff
@@ -14,7 +14,7 @@ clusterNetworks: "10.0.0.0/8,100.64.0.0/10,172.16.0.0/12,192.168.0.0/16"
 # -- Docker image pull policy
 imagePullPolicy: &image_pull_policy IfNotPresent
```

Comment:
Can we remove this anchor now?

Reply:
Similarly, can we remove the `&linkerd_version` anchor?

Reply:
I'm not sure what you mean about the debug container not getting rendered anywhere.  Isn't it rendered from the _debug.tpl partial into the pod patch generated by the proxy injector?  Can't that partial also include the same defaulting logic as everything else?

Reply:
Yep,  You are right. I was under the impression that the injector depends on these values but its similar to others i.e proxy, proxy-init. So, Updated.